ci-operator/step-registry/ipi/conf/mirror/ipi-conf-mirror-commands.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+mkdir -p ${HOME}/.docker
+cp ${CLUSTER_PROFILE_DIR}/pull-secret ${HOME}/.docker/config.json
+oc registry login
+MIRROR_BASE=$( oc get is release -o 'jsonpath={.status.publicDockerImageRepository}' )
+oc adm release new \
+  --from-release ${RELEASE_IMAGE_LATEST} \
+  --to-image ${MIRROR_BASE}-scratch:release \
+  --mirror ${MIRROR_BASE}-scratch \
```

Comment:
These can take a while, so I'd rather guard them with TERM-propagation like [this][1].

[1]: https://github.com/openshift/ci-docs/pull/96#discussion_r576465732